pkgs/applications/networking/irc/catgirl/default.nix

```diff
@@ -0,0 +1,23 @@
+{lib, stdenv, fetchurl, ncurses, libressl, man, pkg-config, ctags}:
+
+stdenv.mkDerivation rec {
+  pname = "catgirl";
+  version = "1.6";
+
+  src = fetchurl {
+    url = "https://git.causal.agency/catgirl/snapshot/${pname}-${version}.tar.gz";
+    sha256 = "0shg02zidqqmvywqqsaazlgg9rd5lhhrvjx6n0lzmdfaawxywciv";
+  };
+
+  nativeBuildInputs = [ pkg-config ctags ];
+  buildInputs = [ ncurses libressl man ];
```

Comment:
```suggestion
  nativeBuildInputs = [ ctags pkg-config ];
  buildInputs = [ libressl man ncurses ];
```

Reply:
oh that makes a lot more sense, i thought you were talking about putting the entire lines in alphabetical order